Libplanet.Benchmarks/Store.cs

```diff
@@ -0,0 +1,147 @@
+        private static readonly int TxsCount;
+        private IStore _store = null;
+
+        static Store()
+        {
+            List<Block<DumbAction>> blocks = new List<Block<DumbAction>>();
+            List<Transaction<DumbAction>> txs = new List<Transaction<DumbAction>>();
+            Block<DumbAction> block = TestUtils.MineGenesis<DumbAction>();
+            blocks.Add(block);
+            PrivateKey key = new PrivateKey();
+            long nonce = 0;
+            for (int i = 0; i < 500; i++)
+            {
+                List<Transaction<DumbAction>> blockTxs = new List<Transaction<DumbAction>>();
+                for (int j = 0; j < i % 5; j++)
+                {
+                    blockTxs.Add(Transaction<DumbAction>.Create(nonce++, key, new DumbAction[0]));
+                }
+                block = TestUtils.MineNext(block, blockTxs);
+                blocks.Add(block);
+                txs.AddRange(blockTxs);
+            }
+            Blocks = blocks.ToImmutableArray();
+            BlockCount = Blocks.Length;
+            Txs = txs.ToImmutableArray();
+            TxsCount = Txs.Length;
+        }
+
+        [IterationSetup(
+            Targets = new []
+            {
+                nameof(PutFirstEmptyBlock),
+                nameof(PutFirstBlockWithTxs),
+                nameof(PutFirstTx),
+            }
+        )]
+        public void InitializeFixture()
+        {
+            var fx = new LiteDBStoreFixture();
+            _store = fx.Store;
+        }
+
+        [Benchmark]
+        public void PutFirstEmptyBlock()
+        {
+            _store.PutBlock(Blocks[0]);
+        }
+
+        [Benchmark]
+        public void PutFirstBlockWithTxs()
+        {
+            _store.PutBlock(Blocks[5]);
+        }
+
+        [IterationSetup(
+            Targets = new []
+            {
+                nameof(PutBlockOnManyBlocks),
+                nameof(GetBlock),
+            }
+        )]
+        public void PutManyBlocks()
+        {
+            InitializeFixture();
+            int i = 0;
+            foreach (Block<DumbAction> block in Blocks)
+            {
+                _store.PutBlock(block);
+                i++;
+                if (i >= Blocks.Length - 1)
+                {
+                    break;
+                }
+            }
+        }
+
+        [Benchmark]
+        public void PutBlockOnManyBlocks()
+        {
+            _store.PutBlock(Blocks[BlockCount - 1]);
+        }
+
+        [Benchmark]
+        public void GetBlock()
+        {
+            _store.GetBlock<DumbAction>(Blocks[BlockCount - 1].Hash);
+        }
+
+        [Benchmark]
+        public void PutFirstTx()
+        {
+            _store.PutTransaction(Txs[0]);
+        }
+
+        [IterationSetup(
+            Targets = new []
+            {
+                nameof(PutTxOnManyTxs),
+                nameof(GetTx),
+            }
+        )]
+        public void PutManyTxs()
```

Comment:
This looks the same as `PutManyBlocks()`. Is it intended?

Reply:
No, you're right.  I'm going to fix this.